### PR TITLE
Change help rendering app to government frontend

### PR DESCRIPTION
**Pull request by mcgoooo**
this supports the template consolidation work

### CODE REVIEW

File: lib/enhancements/edition.rb [hunk 1]
```diff
@@ -44,11 +44,11 @@ class ResurrectionError < RuntimeError
   ].freeze
 
   def self.state_names
-    state_machine.states.map &:name
+    state_machine.states.map(&:name)
```
Comment (whoojemaflip) [inline lib/enhancements/edition.rb]: How does this change relate to the PR?

File: lib/enhancements/edition.rb [hunk 2]
```diff
@@ -44,11 +44,11 @@ class ResurrectionError < RuntimeError
...
   end
 
   scope :internal_search, lambda { |term|
-    regex = Regexp.new(Regexp.escape(term), true)  # case-insensitive
+    regex = Regexp.new(Regexp.escape(term), true) # case-insensitive
```
Comment (whoojemaflip) [inline lib/enhancements/edition.rb]: Same

**Comment by whoojemaflip** [comment]
there's a typo in the commit message

**Comment by mcgoooo** [comment]
@whoojemaflip fixed

File: lib/enhancements/edition.rb (hunk 1 above)
Comment (mcgoooo) [inline lib/enhancements/edition.rb]: it doesn't but the linter is picking up for some unknown reason

**Pull request merged**